zone.js

```diff
@@ -8,7 +8,7 @@ var zone = null;
 function Zone(parentZone, data) {
   var zone = (arguments.length) ? Object.create(parentZone) : this;
 
-  zone.parent = parentZone;
+  zone.parent = parentZone || null;
```

Comment:
Doesn't this invalidate a lot of tests ?
ie `expect(window.zone.parent).toBeDefined();` to test if the zone is the root zone ?
I'll try to update to `isRootZone`